compass/landice/tests/ismip6_forcing/atmosphere/process_smb.py

```diff
@@ -0,0 +1,419 @@
+                "-m", mapping_file,
+                "-v", "smb_anomaly"]
+
+        check_call(args, logger=self.logger)
+
+    def rename_ismip6_smb_to_mali_vars(self, remapped_file_temp, output_file):
+        """
+        Rename variables in the remapped source input data
+        to the ones that MALI uses.
+
+        Parameters
+        ----------
+        remapped_file_temp : str
+            temporary ismip6 data remapped on mali mesh
+
+        output_file : str
+            remapped ismip6 data renamed on mali mesh
+        """
+
+        # open dataset in 20 years chunk
+        ds = xr.open_dataset(remapped_file_temp, chunks=dict(time=20),
+                             engine="netcdf4")
+
+        # build dictionary for ismip6 variables that MALI takes in
+        ismip6_to_mali_dims = dict(
+            time="Time",
+            ncol="nCells")
+        ds = ds.rename(ismip6_to_mali_dims)
+
+        ismip6_to_mali_vars = dict(
+            smb_anomaly="sfcMassBal")
+        ds = ds.rename(ismip6_to_mali_vars)
+
+        # add xtime variable
+        xtime = []
+        for t_index in range(ds.sizes["Time"]):
+            date = ds.Time[t_index]
+            # forcing files do not all match even years, so round up the years
+            # pandas round function does not work for years, so do it manually
+            yr = date.dt.year.values
+            mo = date.dt.month.values
+            dy = date.dt.day.values
+            dec_yr = np.around(yr + (30 * (mo - 1) + dy) / 365.0)
+            date = f"{dec_yr.astype(int)}-01-01_00:00:00".ljust(64)
+            xtime.append(date)
+
+        ds["xtime"] = ("Time", xtime)
+        ds["xtime"] = ds.xtime.astype('S')
+
+        # drop unnecessary variables
+        ds = ds.drop_vars(["lon", "lon_vertices", "lat", "lat_vertices",
+                           "area"])
+
+        # write to a new netCDF file
+        write_netcdf(ds, output_file)
+        ds.close()
+
+    def correct_smb_anomaly_for_climatology(self,
+                                            racmo_clim_file,
+                                            output_clim_ismip6_file,
+                                            output_file_final):
+
+        """
+        Apply the MALI base SMB to the ismip6 SMB anomaly field
+
+        Parameters
+        ----------
+        racmo_clim_file : str
+            RACMO climatology file (1995-2017)
+
+        output_clim_ismip6_file : str
+            remapped and renamed ismip6 climatology file
+
+        output_file_final : str
+             climatology-corrected, final ismip6 smb anomaly file
+        """
+
+        ds = xr.open_dataset(output_file_final)
+        ds_racmo_clim = xr.open_dataset(racmo_clim_file)
+        ds_ismip6_clim = xr.open_dataset(output_clim_ismip6_file)
+
+        # calculate the climatology correction
+        corr_clim = ds_racmo_clim["sfcMassBal"].isel(Time=0) \
+                    - ds_ismip6_clim["sfcMassBal"].isel(Time=0)
```

Comment:
I think the problem may be that it wants the indentation to be 4 more spaces than the previous line, not aligned with the equals sign.  I would handle this by adding parentheses.  This is a widely accepted way of handling this situation.  It is also customary to put the minus sign on the first line, not the second.  The `\` is not needed when you use parentheses:

```suggestion
        corr_clim = (ds_racmo_clim["sfcMassBal"].isel(Time=0) -
                     ds_ismip6_clim["sfcMassBal"].isel(Time=0))
```

And, yes, I know these rules and conventions are frustratingly arbitrary.